packages/edit-post/src/components/sidebar/post-status/index.js

```diff
@@ -4,7 +4,7 @@
 import { __ } from '@wordpress/i18n';
 import { PanelBody } from '@wordpress/components';
 import { withSelect, withDispatch } from '@wordpress/data';
-import { compose } from '@wordpress/compose';
+import { compose, ifCondition } from '@wordpress/compose';
```

Comment:
Fancy. TIL ✨ 

https://github.com/WordPress/gutenberg/blob/da0b1a358a1cf58834c9f48d55b878e0a69167dc/packages/compose/src/higher-order/if-condition/README.md

Reply:
so many neat packages in this repo!